spec/features/users/mygarden/delete_plants_spec.rb

```diff
@@ -9,30 +9,59 @@
                             name: 'gardenthattesting',
                             zip_code: '02300',
                             google_token: ENV['TEST_USER_GOOGLE_TOKEN'],
-                            google_refresh_token: ENV['TEST_USER_GOOGLE_REFRESH_TOKEN']
+                            google_refresh_token: ENV['TEST_USER_GOOGLE_REFRESH_TOKEN'],
+                            calendar_id: '5n59q0ueh4202i4mlstfbqejgc@group.calendar.google.com'
```

Comment:
Adds the `calendar_id` until a better solution is found for testing google calendar related features. By using an existing `calendar` we avoid having to create new one for our test and exceeding the limit from the API